odd-platform-ui/src/redux/interfaces/state.ts

```diff
@@ -158,7 +158,7 @@ export interface DataEntitiesState {
   popular: DataEntityRef[];
   typesDict: {
     types: Dictionary<DataEntityType>;
-    subtypes: Dictionary<DataEntitySubType>;
+    entityClasses: Dictionary<DataEntityClass>;
   };
```

Comment:
I think that now it's not only `typesDict`, but `classAndTypeDict`.
And I suggest using the same naming - `entityTypes` and `entityClasses`.

Reply:
done